packages/core/src/tools/heavyCustomerDataWarning.ts

```diff
@@ -13,12 +13,19 @@ export const enum CustomerDataType {
   LoggerContext = 'logger context',
 }
 
+let alreadyWarned: { [key: string]: boolean } = {}
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**❓ question:** ‏FMU, for a npm setup with both logs and rum, it would mean that we would warn customers once for all SDK instances, is it intentional?

Reply:
No it was not :) 
I moved the "already warn check" to `contextManager` and `featureFlagContext` to solve it. 
Let me know what you think.